podcasts/PlaybackManager.swift

```diff
@@ -2347,4 +2347,28 @@ extension PlaybackManager {
     }
 }
 
+// MARK: - SearchResults
+extension PlaybackManager {
+
+    func playEpisodeSearchResult(_ searchEpisode: EpisodeSearchResult, firstTry: Bool = true) {
+        let dataManager = DataManager.sharedManager
+
+        // Get the bookmark's BaseEpisode so we can load it
+        guard let episode = dataManager.findBaseEpisode(uuid: searchEpisode.uuid) else {
+            if firstTry {
```

Comment:
Can this be a guard and return early?

Reply:
Updated here: [fefac09](https://github.com/Automattic/pocket-casts-ios/pull/3518/commits/fefac093c092b6c1c3c092093c6316a934dba266)